lib/telemetry_loggers/live_view_logger.ex

```diff
@@ -0,0 +1,93 @@
+defmodule TelemetryLoggers.LiveViewLogger do
+  @moduledoc """
+  Logs one log per request on a LiveView call.
+
+  It includes the following metadata:
+  - `callback` - Which kind of live view event is listening on, eg: `:mount`, `:handle_params`, etc...
+  - `kind` - The source of the event, eg: `live_view` or `live_component`
+  - `duration`: The duration in microseconds
```

Comment:
Since the metadata changes per event type, maybe we should document it properly, maybe like a list of "source" (better name to be defined) and callback and what are the metadata available (we can still have a list of the metadata that all event includes, something like (rough draft):

The metadata it includes will depend on the source of the event (either a LiveView or a LiveComponent) and on the callback, but some metadata will be available on all logs.

- Available on all logs:
  - `callback` - ...
  - `source` - ...
  - `duration_us` - ...
  - ...
- Available when `callback` is `:handle_event`:
  - `:event` - The event being handled...
- Available when `callback` is either `:handle_params` or `:mount`:
  - ...
- Available when `source` is ``:live_component`:
  - `:component` - The name of the component module.

Also, here it says `:kind` but we are actually using the key `:source`.